docker/serverless/env-start-esp.py

```diff
@@ -34,13 +34,52 @@
 # The command being run must be the 0th arg.
 ARGS = [CMD, "--enable_backend_routing"]
 
+# A set of ESP flags supported for ESP Serverless
+SUPPORTED_ESP_FLAGS = set([
+  "--cors_allow_credentials",
+  "--cors_allow_headers",
+  "--cors_allow_methods",
+  "--cors_allow_origin_regex",
+  "--cors_expose_headers",
+  "--enable_debug",
+  "--service_control_url_override"
+])
```

Comment:
`--management` needs to be supported also.

Reply:
Done